packages/front-end/components/Features/RuleStatusPill.tsx

```diff
@@ -0,0 +1,70 @@
+import { FeatureRule, ScheduleRule } from "back-end/types/feature";
+import React from "react";
+
+type Props = {
+  rule: FeatureRule;
+  upcomingScheduleRule: ScheduleRule | null;
+  currentScheduleStatus: "enabled" | "disabled";
+};
+
+export default function RuleStatusPill({
+  rule,
+  upcomingScheduleRule,
+  currentScheduleStatus,
+}: Props) {
+  if (!rule.enabled) {
```

Comment:
Rather than having all of these blocks, can I create a variable `pillText` and build a switch/case statement and then just feed that into the return?